docs/usage/upgrade_from_pre-0.2.0.rst

```diff
@@ -0,0 +1,71 @@
+Upgrading from MetalK8s < 0.2.0

```

Comment:
I'd like to see this removed from the documentation altogether. This 'upgrade' was never supported in the first place, new deployments shouldn't be using 0.2.0, and any existing 0.2.0 deployments have *or* been upgraded, *or* the owner can figure out how to upgrade by reading old versions of the documentation.